tests/unit/multiclass/BaggingMachine_unittest.cc

```diff
@@ -27,11 +30,168 @@
 using namespace shogun;
 using ::testing::Return;
 
+class BaggingMachine : public ::testing::Test
+{
+public:
+	CDenseFeatures<float64_t>* features_test;
+	CDenseFeatures<float64_t>* features_train;
+	CMulticlassLabels* labels_train;
+
+	SGVector<bool> ft;
+	virtual void SetUp()
+	{
+		sg_rand->set_seed(1);
+		generate_toy_data_weather();
+	}
+
+	virtual void TearDown()
+	{
+		SG_UNREF(features_train);
+		SG_UNREF(features_test);
+		SG_UNREF(labels_train);
+	}
+
+	void generate_toy_data_weather()
```

Comment:
yeah you can use the utils part to add these type of generators/utility functions